python/cuml/linear_model/logistic_regression_mg.pyx

```diff
@@ -0,0 +1,198 @@
+#
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# distutils: language = c++
+
+from cuml.internals.safe_imports import gpu_only_import
+cp = gpu_only_import('cupy')
+from cuml.internals.safe_imports import cpu_only_import
+np = cpu_only_import('numpy')
+
+from libcpp cimport bool
+from libc.stdint cimport uintptr_t
+
+from cuml.common import input_to_cuml_array
+import numpy as np
+
+from cuml.internals.array import CumlArray
+from cuml.internals.api_decorators import device_interop_preparation
+from cuml.linear_model import LogisticRegression 
+from cuml.solvers.qn import QN
+from cuml.solvers.qn import QNParams
+
+from pylibraft.common.handle cimport handle_t
+
+from cuml.solvers.qn import __is_col_major
+
+# the cdef was copied from cuml.linear_model.qn
+cdef extern from "cuml/linear_model/glm.hpp" namespace "ML::GLM" nogil:
+
+    cdef enum qn_loss_type "ML::GLM::qn_loss_type":
```

Comment:
Since this isn't MG specific, we should centralize it so the SG and MG can use the same one.

Reply:
This will affect single-GPU logistic_regression.pyx. Is it ok to do it in the next PR to avoid this one being too long?

Reply:
I agree with the logic of adding this change in a follow up, but could you open a GH issue capturing this (and any other small) follow up items?

Reply:
Also, can you add a little todo here and post the number of the github issue? This helps us map it back to the code so it doesn't get lost/forgotten.